pom.xml

```diff
@@ -55,7 +55,7 @@
     <dep.junit.version>4.13.2</dep.junit.version>
     <dep.logback.version>1.2.11</dep.logback.version>
     <!-- seems to have an impact with java 11 integration, update with caution -->
-    <dep.maven-surefire-plugin.version>3.0.0-M3</dep.maven-surefire-plugin.version>
+    <dep.maven-surefire-plugin.version>3.0.0-M8</dep.maven-surefire-plugin.version>
```

Comment:
😬

Reply:
:(

Reply:
In my defense that comment came after I created this PR back in January :)